burnman/tools.py

```diff
@@ -643,3 +644,109 @@ def check_eos_consistency(m, P=1.e9, T=300., tol=0.01, verbose=False):
             print('Not satisfied all EoS consistency constraints for {0:s}'.format(m.to_string()))
             
     return consistency
+
+
+def _pad_ndarray_inverse_mirror(array, padding):
```

Comment:
It's true that the filter runs along each axis separately, but that means that the corners *are* used. Take this example, where there is a 1 cell padding all the way round the array and I take the filter to be (0.25*(values either side) + 0.5*value):

1   2 2   1             7/4   7/4 
1 | 3 3 | 1  -->   | 10/4  10/4  |  ->  | 10/4  10/4 |
1 | 4 4 | 1         |  13/4  13/4 |       | 13/4   13/4 |
1   5 5   1            16/4  16/4 

The corner padding is used in the first step.